src/widgets/helper/ChannelView.hpp

```diff
@@ -208,6 +227,12 @@ class ChannelView final : public BaseWidget
     void enableScrolling(const QPointF &scrollStart);
     void disableScrolling();
 
+    /**
+     * @param layout Must be from this channel.
+     * @param messageIdx Must be an index into this channel.
+     */
+    void scrollToMessage(MessageLayout *layout, size_t messageIdx);
```

Comment:
Rename to something more internally appropriate, maybe something like `scrollToMessageInLayout`

Reply:
Done in c8325c32479116991f940b240be69706344abac0.